.github/knip.ts

```diff
@@ -1,11 +1,12 @@
 import type { KnipConfig } from "knip";
 
 const config: KnipConfig = {
-  entry: ["src/worker.ts"],
+  entry: ["src/worker.ts", "deploy/setup-kv-namespace.ts"],
   project: ["src/**/*.ts"],
+  ignore: ["jest.config.ts"],
   ignoreBinaries: ["i"],
   ignoreExportsUsedInFile: true,
-  ignoreDependencies: ["@mswjs/data", "esbuild", "eslint-config-prettier", "eslint-plugin-prettier", "msw"],
+  ignoreDependencies: ["@mswjs/data", "esbuild", "eslint-config-prettier", "eslint-plugin-prettier", "msw", "ts-node"],
```

Comment:
Use `tsx` not `ts-node`

Reply:
`ts-node` is required for `jest` to accept `.ts` configuration.